### PR TITLE
feat: 2022-10-07 storage release notes

### PR DESCRIPTION
**Pull request by kelvich**
Deploy preview:
https://websitemain62807-relnotes1710222.gtsb.io/docs/release-notes/

### CODE REVIEW

File: content/release-notes/2022-10-07.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+label: 'Storage'
+---
+
+### What's new
+
+- Pageserver: Increase default compaction_period setting to 20 s
```
Comment (danieltprice) [inline content/release-notes/2022-10-07.md:7]:
```suggestion
- Pageserver: Increased the default compaction_period setting to 20 seconds, affecting the per-tenant compaction background process. With numerous projects, the previous default setting of 1 second caused excessive CPU usage.
```

File: content/release-notes/2022-10-07.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+- Pageserver: Initial support for online tenant rebalancing
```
Comment (danieltprice) [inline content/release-notes/2022-10-07.md:8]:
```suggestion
- Pageserver: Added initial support for online tenant rebalancing.
```

File: content/release-notes/2022-10-07.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+- Pageserver: Support for multiple postgres versions (14 and 15 for now)
```
Comment (danieltprice) [inline content/release-notes/2022-10-07.md:9]:
```suggestion
- Pageserver: Added support for multiple PostgreSQL versions.
```

File: content/release-notes/2022-10-07.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+- Compute: Add `h3_pg` extension
```
Comment (danieltprice) [inline content/release-notes/2022-10-07.md:10]:
```suggestion
- Compute: Added support for the `h3_pg` extension, which provides PostgreSQL bindings for the [H3 Core Library](https://github.com/uber/h3).
```

File: content/release-notes/2022-10-07.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+- Compute: Beta version of sequential scan page prefetch
+- Compute: Backpressure algorithm tuning
+- Proxy: Improve error messages and logging
```
Comment (danieltprice) [inline content/release-notes/2022-10-07.md:13]:
```suggestion
- Proxy: Improved error messages and logging.
```

File: content/release-notes/2022-10-07.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+---
+
+### What's new
+
+- Pageserver: Increase default compaction_period setting to 20 s
+- Pageserver: Initial support for online tenant rebalancing
+- Pageserver: Support for multiple postgres versions (14 and 15 for now)
+- Compute: Add `h3_pg` extension
+- Compute: Beta version of sequential scan page prefetch
+- Compute: Backpressure algorithm tuning
```
Comment (danieltprice) [inline content/release-notes/2022-10-07.md:12]: Tuned the backpressure algorithm, which manages the lag between the Pageserver and compute node or the Pageserver and the Write-ahead Log (WAL) service.

Comment (danieltprice) [inline content/release-notes/2022-10-07.md:12]:
```suggestion
- Compute: Tuned the backpressure algorithm, which manages the lag between the Pageserver and compute node or the Pageserver and the Write-ahead Log (WAL) service.
```

File: content/release-notes/2022-10-07.md [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+label: 'Storage'
+---
+
+### What's new
+
+- Pageserver: Increase default compaction_period setting to 20 s
+- Pageserver: Initial support for online tenant rebalancing
+- Pageserver: Support for multiple postgres versions (14 and 15 for now)
+- Compute: Add `h3_pg` extension
+- Compute: Beta version of sequential scan page prefetch
```
Comment (danieltprice) [inline content/release-notes/2022-10-07.md:11]:
```suggestion
- Compute: Introduced a beta version of sequential scan page prefetch, which improves sequential scan performance.
```

File: content/release-notes/2022-10-07.md (hunk 3 above)
Comment (danieltprice) [reply content/release-notes/2022-10-07.md:9]: Do we want to announce support for PG 15 here?

File: content/release-notes/2022-10-07.md (hunk 7 above)
Comment (danieltprice) [reply content/release-notes/2022-10-07.md:11]: Is the above addition accurate?

File: content/release-notes/2022-10-07.md (hunk 3 above)
Comment (kelvich) [reply content/release-notes/2022-10-07.md:9]: +1, let's keep it the way you've phrased it

File: content/release-notes/2022-10-07.md (hunk 7 above)
Comment (kelvich) [reply content/release-notes/2022-10-07.md:11]: It is, thanks!

Comment (stepashka) [reply content/release-notes/2022-10-07.md:11]: experimental? early? not sure about beta.
also, is the prefetch enabled in production? this entry looks like it is. is it actually isn't smth like 'added support for experimental sequential scan prefetch implementation' may be more appropriate.

Comment (stepashka) [reply content/release-notes/2022-10-07.md:11]: i am also thinking about the the readers who aren't familiar with db perf tuning
are there any links we can add here to help them relate to sequential scan performance?

File: content/release-notes/2022-10-07.md (hunk 1 above)
Comment (stepashka) [reply content/release-notes/2022-10-07.md:7]: besides the increased CPU usage on pageserver that is not necessarily affecting the users, were there symptoms that users could feel that may be relieved with this?

File: content/release-notes/2022-10-07.md (hunk 6 above)
Comment (stepashka) [reply content/release-notes/2022-10-07.md:12]: should we add 'backpressure' to glossary?
what are the symptoms that this particular adjustment improves or relieves?

**Review by stepashka — COMMENTED** [review]
added some questions and suggestions

File: content/release-notes/2022-10-07.md (hunk 7 above)
Comment (danieltprice) [reply content/release-notes/2022-10-07.md:11]: How about: 
"Added support for a future implementation of sequential scan prefetch, which improves I/O performance for operations such as table scans."

File: content/release-notes/2022-10-07.md (hunk 1 above)
Comment (danieltprice) [reply content/release-notes/2022-10-07.md]: Looks like the issue was found in captest. 
https://github.com/neondatabase/neon/issues/2542
https://github.com/neondatabase/neon/pull/2527
I have revised the entry with more details.

File: content/release-notes/2022-10-07.md (hunk 2 above)
Comment (danieltprice) [reply content/release-notes/2022-10-07.md]: changed to relocation.

File: content/release-notes/2022-10-07.md (hunk 7 above)
Comment (danieltprice) [reply content/release-notes/2022-10-07.md]: Revised as mentioned above.

File: content/release-notes/2022-10-07.md (hunk 6 above)
Comment (danieltprice) [reply content/release-notes/2022-10-07.md]: Created a PR for "backpressure" glossary addition.
Updated the release note entry, which was reviewed by @knizhnik.

**Review by stepashka — APPROVED** [review]

**Pull request merged**